src/nashpy/algorithms/lemke_howson.py

```diff
@@ -3,15 +3,16 @@
 from itertools import cycle
 
 import numpy as np
-
+import numpy.typing as npt
+from typing import Tuple, Set, Iterable
 from nashpy.integer_pivoting import (
     make_tableau,
     non_basic_variables,
     pivot_tableau,
 )
 
 
-def shift_tableau(tableau, shape):
+def shift_tableau(tableau: npt.NDArray, shape: Tuple[int, ...]) -> npt.NDArray:
```

Comment:
Can this be:

```suggestion
def shift_tableau(tableau: npt.NDArray, shape: Tuple[int, int]) -> npt.NDArray:
```

Reply:
Actually come to think about it I don't think that's a good suggestion. The type hint you have chosen is great because it specifies that the only thing needed is an `int` at the first position :+1: